MoreLinq.Test/TestException.cs

```diff
@@ -21,5 +21,9 @@ namespace MoreLinq.Test
     /// Reserved for use within tests.
     /// </summary>
 
-    sealed class TestException : System.Exception {}
+    sealed class TestException : System.Exception
```

Comment:
Revert these changes because they're not needed (based on my earlier suggestion to use `Exception.Data` instead).